sherpa/models/regrid.py

```diff
@@ -311,13 +244,12 @@ def overlaps(self, other):
         return self.x_axis.overlaps(other.x_axis)
 
     def __contains__(self, other):
-        """
-        check if this space properly contains the `other` space, i.e. if the `other` space is contained
-        within the boundaries of `self`.
+        """Is `other` contained in the boundaries of this space?
```

Comment:
Question: Does "contain" mean that start/stop of other are within the start/stop boundaries of the self spaceor does it mean that the exact numbers are part of it. On other words, is [3,4,5] contained in [2.5, 3.5, 4.5, 5.5] or not?

Reply:
I don't know. I didn't write this code and I do not understand it.

Reply:
It's just checking the start/stop boundaries, not the actual values in the middle. (Just look at the only line of code in this function). I should have expanded that before I wrote the comment... here is suggested text:
"Are all elements of `other` within the low end to high end range of this space?"

Reply:
I've changed this - it's a dunder method so I don't think most people will see it!